support/supportedversion/version.go

```diff
@@ -15,7 +15,7 @@ var MinSupportedVersion = semver.MustParse(subtractMinor(&LatestSupportedVersion
 
 // SupportedPreviousMinorVersions is the number of minor versions prior to current
 // version that are supported.
-const SupportedPreviousMinorVersions = 2
+const SupportedPreviousMinorVersions = 3
```

Comment:
Our generic generic policy is N-2. Can we make this IBM specific in the IsValidReleaseVersion logic?

Reply:
We can!